config/kubernetes/sig-release/teams.yaml

```diff
@@ -329,59 +329,59 @@ teams:
             description: Members of the CI Signal team for the current release
               cycle.
             members:
+            - mansikulkarni96 # 1.29 CI Signal Shadow
+            - pacoxu # 1.29 CI Signal Shadow
+            - reetasingh # 1.29 CI Signal Shadow
+            - rjsadow # 1.29 CI Signal Shadow
```

Comment:
@Priyankasaggu11929 You missed John Anderson ([@sontek](https://github.com/sontek))

Reply:
Thanks for catching. Updated it in recent commit refresh!